HackIllinois/Controllers/ViewControllers/HICountdownViewController.swift

```diff
@@ -126,8 +123,27 @@ extension HICountdownViewController {
 }
 
 extension HICountdownViewController {
+    @objc func handleApplicationDidBecomeActive(notification: Notification) {
+        updateTimeDifference()
+        setupCounters()
+    }
+
+    func setupCounters() {
+        hourFrame = hoursRemaining * FRAMES_PER_TICK
+        minuteFrame = minutesRemaining * FRAMES_PER_TICK
+        secondFrame = secondsRemaining * FRAMES_PER_TICK
+
+        hours.setProgress(frame: hourFrame)
+        minutes.setProgress(frame: minuteFrame)
+        seconds.setProgress(frame: secondFrame)
+    }
+
     @objc func updateCountdown() {
         updateTimeDifference()
+        guard timeDifference > 0 else {
```

Comment:
probably shouldn't invalidate the counter, if countdownDate is updated it the UI won't automatically change